modules/core/assets/ships/imperialCapital/imperialCapital.json

```diff
@@ -0,0 +1,443 @@
+{
+    "size": 3.5,
+    "maxLife": 800,
+    "type": "big",
+    "price": 20000,
+    "e1Pos": "0.08 0.37",
+    "e2Pos": "0.08 0.63",
+    "displayName": "Imperial Capital Ship",
```

Comment:
Can't we simply name it as `Imperial Capital` ? I think the Ship might be kinda redundant here.